system-variables.md

```diff
@@ -742,9 +742,17 @@ Constraint checking is always performed in place for pessimistic transactions (d
 ### tidb_enable_index_merge <span class="version-mark">New in v4.0</span>
 
 - Scope: SESSION | GLOBAL
-- Default value: `OFF`
+- Default value: `ON`
 - This variable is used to control whether to enable the index merge feature.
 
+> **Note:**
+>
+> - When upgrading a TiDB cluster from versions earlier than v4.0.0 to v5.4.0 or later, this variable is disabled by default to prevent performance regression due to the changes of an execution plan.
```

Comment:
```suggestion
> - After upgrading a TiDB cluster from versions earlier than v4.0.0 to v5.4.0 or later, this variable is disabled by default to prevent performance regression due to changes of execution plans.
```